app/backend/gwells/settings/__init__.py

```diff
@@ -266,7 +266,7 @@
     'DEFAULT': {
         'CACHE': not DEBUG,
         'BUNDLE_DIR_NAME': 'bundles/',  # must end with slash
-        'STATS_FILE': os.path.join(BASE_DIR, 'frontend/webpack-stats.json')
+        'STATS_FILE': os.path.join(BASE_DIR, '../frontend/webpack-stats.json')
```

Comment:
i wonder if we can put this file into the backend dir?

Reply:
Yes, but I think we should do that as part of cleaning up the django/vue build dependancy.